config/packages/dev/monolog.yaml

```diff
@@ -1,18 +1,28 @@
+services:
+    Monolog\Processor\WebProcessor:
+        tags:
+            - { name: monolog.processor }
+
+    Monolog\Processor\IntrospectionProcessor:
+        tags:
+            - { name: monolog.processor }
+
+    monolog.formatter.dev:
+        class: Monolog\Formatter\LineFormatter
+        arguments:
+            - " >    %%datetime%% [%%level_name%%] %%message%%\n >>   %%extra.class%%:%%extra.function%%:%%extra.line%%\n >>>  %%extra.http_method%% %%extra.url%%\n >>>> %%context%%\n\n"
+            - "H:i:s"
+            - false
+            - false
+
 monolog:
     handlers:
         main:
             type: stream
             path: "%kernel.logs_dir%/%kernel.environment%.log"
```

Comment:
instead of kerne_dir should it be php://stdout in order to be docker compliant ?

Reply:
updated